src/components/SortDropdown.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  hiddenStyle: {
+    display: 'none',
+  },
+  selectedOptionTextStyle: {
+    fontWeight: '500',
+    fontFamily: 'Work Sans',
+    fontSize: '14px',
+    lineHeight: '18px',
+    margin: 'auto 0 auto 0',
+  },
+  optionTextStyle: {
+    fontWeight: '500',
+    fontFamily: 'Work Sans',
+    fontSize: '14px',
+    lineHeight: '18px',
+    margin: 'auto 0 auto 10px',
+  },
+  dropdown: {
+    display: 'flex',
+    cursor: 'pointer',
+    backgroundColor: theme.palette.background.default,
+    boxSizing: 'border-box',
+    borderRadius: '4px 4px 0px 0px',
+    minHeight: '43px',
+    minWidth: '177px',
+    border: '1px solid #BCBCBC',
+  },
+  selectedOptionGridStyle: {
+    display: 'inline-flex',
+    marginLeft: '10px',
+  },
+  dropdownIconStyle: {
+    marginLeft: '10px',
+    wdith: '32px',
```

Comment:
Change 'wdith' to 'width'
```suggestion
    width: '32px',
```